lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -51,10 +51,37 @@
       }).insertAfter(textarea);
       textarea.css('display', 'none');
 
+      // Create a div for the autosave status message
+      var autoSaveMessage = document.createElement('div');
+      $(autoSaveMessage).addClass('div position-fixed bottom-0 left-0');
+      $('body').append(autoSaveMessage);
+
+      var autoSaveTimer = null;
+
       // NOTE: This requires the page to have only one 'gollum-editor-body'.
       var editor = ace.edit(editDiv[0], {rtlText: true});
       window.ace_editor = editor;
 
+      // Check to see if we have any autosaved text and show a message to
+      // restore it if present.
+      var storageKey = 'gollum_autorecover_' + window.location;
+      var savedText = localStorage.getItem(storageKey);
+
+      if (savedText) {
```

Comment:
Ideally, I'd like to have this `div` and the `button` in the editor template as well, and just activate/unhide it when relevant. That way we don't have these CSS classes added in the JS.

Reply:
That makes sense. I'll make the appropriate changes and push an update.

Reply:
Thanks!